js/models/conversations.js

```diff
@@ -1230,18 +1243,26 @@
       if (!thumbnail) {
         return false;
       }
-      const thumbnailWithData = await loadAttachmentData(thumbnail);
-      thumbnailWithData.objectUrl = this.makeObjectUrl(
-        thumbnailWithData.data,
-        thumbnailWithData.contentType
-      );
+      try {
+        const thumbnailWithData = await loadAttachmentData(thumbnail);
+        thumbnailWithData.objectUrl = this.makeObjectUrl(
```

Comment:
Minor: We could now use `Signal.Util.arrayBufferToURL`.

Reply:
Will change it. But I'll say that one thing I don't like about the hiding of the `URL.createObjectURL` call inside of `Signal.Util.arrayBufferToURL` is that you still have to do a `URL.revokeObjectURL` yourself afterwards. There symmetry is missing.